babel.config.main.js

```diff
@@ -8,6 +9,7 @@ const plugins = [
 
 // allows dynamic `import()` in Node tests.
 if (process.env.NODE_ENV === 'test') {
+  plugins.splice(0, 1); // Remove 'babel-plugin-react-compiler'
```

Comment:
Probably don't do this. Jest may need updated itself with react-compiler support.